release_notes.md

```diff
@@ -3,18 +3,18 @@
 ## Enhancement
 
 - Introduced New playbook:
-    - **Ingest Latest CISA ICS Advisory**, playbook ingests all the ICS Advisory record.
-    - **Ingest Latest CISA ICS Medical Advisory**, playbook ingests all the ICS Medical Advisory record.
+    - **Ingest CISA ICS Advisory**, playbook ingests all the ICS Advisory record.
+    - **Ingest CISA ICS Medical Advisory**, playbook ingests all the ICS Medical Advisory record.
     - **Notify for SLA and Report**, playbook notify for BOD 22-01 KEV Alert SLA remaining time and report.
     - **> Check KEV Alert SLA**, playbook check for KEV Alert SLA to notify.
 
 - Introduced New Schedule:
-    - **Ingest Latest ICS Advisory**, triggered daily to ingest ICS Advisory
-    - **Ingest Latest ICS Medical Advisory**, triggered daily to ingest ICS Medical Advisory
+    - **Ingest CISA ICS Advisory**, triggered daily to ingest ICS Advisory
+    - **Ingest CISA ICS Medical Advisory**, triggered daily to ingest ICS Medical Advisory
 
 - Introduced New field:
     - `SSVC`, `Closed On`, and `Response Due Date` in **KEV Alert** module.
-    - `Advisory Type` in **ICS Advisory** moduel and picklist `Advisory Type` for same.
+    - `Advisory Type` in **ICS Advisory** module and picklist `Advisory Type` for same.
 
 - Introduced New connector:
```

Comment:
Having "Introduced" in every section, can we have one section which states all New introduced stuff?